src/main/java/com/example/auctionapp/api/AuthController.java

```diff
@@ -29,4 +35,14 @@
         return authenticationService.register(registerRequest);
     }
 
+    @PutMapping("/update")
+    public ResponseEntity<?> update(@RequestBody UpdateRequest updateRequest) {
+        return userService.updateUser(updateRequest);
+    }
+
+    @PutMapping("/deactivate")
+    public ResponseEntity<?> deactivate(HttpServletRequest httpServletRequest) {
+        return userService.deactivateAccount(httpServletRequest);
+    }
```

Comment:
Since those two endpoints call UserService, it might be more appropriate to put them in userController. And have userService on UI as well